translations/pl.json

```diff
@@ -1,195 +1,664 @@
 {
-    "confirm": "Potwierdź",
-    "phone-national": "Krajowy",
+    "about-app": "O aplikacji",
+    "account-generation-message": "Daj mi chwilkę, muszę wykonać szalone obliczenia, żeby utworzyć dla Ciebie konto!",
+    "active-online": "Online",
+    "active-unknown": "Nieznany",
+    "add": "Dodaj",
+    "add-a-status": "Dodaj status...",
+    "add-bootnode": "Dodaj bootnode",
+    "add-existing-account": "Dodaj istniejące konto",
+    "add-json-file": "Dodaj plik JSON",
+    "add-mailserver": "Dodaj Mailserver",
+    "add-members": "Dodaj użytkowników",
+    "add-network": "Dodaj sieć",
+    "add-new-contact": "Dodaj nowy kontakt",
+    "add-new-network": "Dodaj nową sieć",
+    "add-to-contacts": "Dodaj do kontaktów",
+    "add-wnode": "Dodaj mailserver",
+    "address": "Adres",
+    "address-explication": "Być może tutaj powinien znajdować się tekst wyjaśniający, czym jest adres i gdzie go szukać",
+    "advanced-settings": "Zaawansowane ustawienia",
+    "agree-by-continuing": "Kontynuując, zgadzasz się na naszą ",
+    "all": "Wszystko",
+    "allow": "Pozwól",
+    "allowing-authorizes-this-dapp": "Zgadając się, umożliwisz tej aplikacji DApp na pobranie adresu Twojego portfela i uruchomieniu Web3 ",
+    "already-have-account": "Posiadam już konto",
     "amount": "Kwota",
-    "members-active": {
-        "one": "1 użytkownik, 1 aktywny",
-        "other": "Użytkownicy:{{count}}, aktywni:{{count}}",
-        "zero": "brak użytkowników"
-    },
+    "amount-placeholder": "Określ kwotę…",
```

Comment:
@chadyj "..." looks weird here

Reply:
The "..." is in Lokalise like that.

Reply:
Yeah, I kept the '…' from English language. Should we remove them @lukaszfryc ?

Reply:
no, we can leave it as it is